src/oop/object_independence/medium/q4/MainTest.java

```diff
@@ -0,0 +1,179 @@
+package oop.object_independence.medium.q4;
+
+import global.ObjectTest;
+import global.variables.Clause;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertArrayEquals;
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest {
+    // Java
```

Comment:
```suggestion
    // Parsons
```

Reply:
I this comment https://github.com/canvas-gamification/canvas-gamification-junit-tests/pull/674#discussion_r1213811686 I asked you to fix the formatting on the Parsons comment, but you switched it to Java. Is there another reason for the change?

Reply:
We decided that medium questions should be partially given java